### DISCUSSION TITLE
[ARM64 release] Does moosefs team have plan to support more?

### DISCUSSION THREAD

**Discussion by bzhaoopenstack** [moosefs/moosefs#405] [General]
Hi team,

we have strongly requests to make moosefs support arm64 platform. So I search in moosefs community and try to get some helps from upstream.

What I have found is that there is already a official arm64 community version release on https://moosefs.com/download/#current, but only for ubuntu 20.04 . That's a great news for us. 

So, does team have plan to support more OS? Such as supporting debian and yum arm64 packages release?

Thank you very much.

**Comment by onlyjob** [comment]
_MooseFS_ is available from Debian: https://tracker.debian.org/pkg/moosefs and it builds on all supported architectures: https://buildd.debian.org/status/package.php?p=moosefs

I recommend to stay away from general purpose derivatives (especially Ubuntu) unless you know what you are doing.

**Comment by chogata** [comment]
We cannot at this point say what packages we will provide for arm64 in the future. However, there is always the option of compiling MooseFS from source and as far as we know, if it worked on intel and a certain OS/distribution of OS, it should also work on arm64 and the same OS/distribution. We welcome any reports about problems compiling MooseFS on arm64.
